Bulldozer.CSV/Maps/GroupMember.cs

```diff
@@ -462,15 +464,15 @@ private static void AddMissingRelationshipGroups()
                         var groupMember = new GroupMember();
                         groupMember.PersonId = personId;
                         groupMember.GroupRoleId = ownerRoleId.Value;
+                        groupMember.GroupTypeId = relationshipGroupType.Id;
 
                         var group = new Group();
                         group.Name = relationshipGroupType.Name;
                         group.Guid = Guid.NewGuid();
                         group.GroupTypeId = relationshipGroupType.Id;
 
-                        groupGroupMembersToInsert.Add( group.Guid, groupMember );
-
                         groupsToInsert.Add( group );
+                        groupGroupMembersToInsert.Add( group.Guid, groupMember );
```

Comment:
This change didn't get reverted, was it actually necessary?

```suggestion
                        groupGroupMembersToInsert.Add( group.Guid, groupMember );

                        groupsToInsert.Add( group );
```